docs/getting-started/tutorials/backoffice/auth0-integration.mdx

```diff
@@ -0,0 +1,756 @@
+7. Repeat the first 6 steps to add the `user-manager-service` configurations page, choosing your favourite *Icon* and with the following changes:
+```
+Id: ums-configurations
+Label: Configurations
+```
+
+Then you have to set the user menu:
+1. Click on the `Layout` tab.
+2. Select `Layout` from the left-side menu.
+3. Click on the `Edit property` button under the `Login and user context menu` section on the right-side menu, under the `Advanced Properties`.
+4. On the opened modal click fill the `User Context URL` with the value `/userinfo`
+5. Click on the `Add property` button related to `Logout` section.
+6. Fill the `Redirect Url` with `/logout?redirect=/web-login?redirect=/backoffice`.
+7. Click on the `Save` button.
+
+## Configure the authentication process
+
+In order to protect your Backoffice you need to follow these steps:
+1. Go to Endpoints, select `/backoffice` and check the `Authentication required` checkbox under the `Endpoint settings`
+2. Set the following variables on `Authentication Service`:
+  - `USERINFO_URL`: http://user-manager-service/userinfo
+  - `AUTHORIZATION_HEADERS_TO_PROXY`: cookie,authorization,client-type
+
+The final configuration step is to change the API key client type with the one defined as defaultClient in the Auth0 Client configuration.
+
+:::info
+This step will allow the oauth-login-site microservice to obtain the right client key necessary for the authorization process.
+:::
+
+In our case, the client type will be **frontEnd**. The final configuration will look something like this: 
+
+![API Key](./img/auth0_06-api-key.png)
+
+After that, you have finally completed the configuration of the authentication process.
+
+## Save and Deploy
+
+With all the configurations in place, save your changes and then deploy. For further details on the deploy command, consult the [Console Handbook](/getting-started/handbooks/project/usage.md).
+
+![Backoffice Homepage](img/auth0_07-homepage.png)
+
+## Create a user using Backoffice
+In order to create your first user you need to create a configuration. To do that:
+1. Click on `Configurations` page.
+2. Click on `Add new` button.
+3. Fill the form with the following parameters:
+![Backoffice Page Composer](img/auth0_08-create-config.png)
+4. Click on `Create` button and `Confirm` on the warning modal.
+
+Now it's possible to create the first user:
+1. Click on `Users` page.
+2. Click on `Add new` button.
+3. Fill the form with the `Name` and `Email` of your first user and using `user` as `Group`.
+4. Click on `Create` button and `Confirm` on the warning modal.
+
+Congratulations! You have successfully created your first user using the backoffice.
+If you try to access the Backoffice with this new user you won't see the users management pages that are protected with the ACLs created in the previous steps.
+Now if you delete the created user it will be blocked on Auth0 and marked as `TRASH` inside the CRUD collection.
+
+## Backoffice Tutorials
```

Comment:
I would add a sectoin where we actually protect some pages/component using ACLs

Reply:
I've added a section inside **Configure the Backoffice application** because the ACLs are added during the creation of new pages